clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -143,6 +143,15 @@ const Formula &getFormula(const ValueDecl &D, const Environment &Env) {
   return cast<BoolValue>(Env.getValue(D))->formula();
 }
 
+const BindingDecl *findBindingDecl(const char *Name, ASTContext &ASTCtx) {
```

Comment:
why place this parameter second (vs first, like findValueDecl)? I don't have a preference, just curious about the difference.

Reply:
Not quite sure why I did this, thanks for the catch :)